src/mpi/mpi_caf.c

```diff
@@ -584,8 +585,6 @@ PREFIX (register) (size_t size, caf_register_t type, caf_token_t *token,
 
   if (type != CAF_REGTYPE_COARRAY_ALLOC_REGISTER_ONLY)
     {
-//      PREFIX(sync_all) (NULL,NULL,0);
-
```

Comment:
commit https://github.com/sourceryinstitute/opencoarrays/pull/261/commits/7e917b2d429306114f4c80b4ce723906bea27c8f to be precise